core/src/main/java/com/scalar/db/storage/jdbc/query/PreparedStatementBinder.java

```diff
@@ -133,6 +134,19 @@ public void visit(BlobColumn column) {
     }
   }
 
+  public void bindLikeClause(LikeExpression likeExpression) {
```

Comment:
Hmm, we can't use the visitor pattern only for the LIKE operation. Honestly, I don't find it ideal, but I'm out of alternatives, and maybe it's acceptable...

Reply:
Yeah, we have the same feelings...